test/integration/controller/core/workload_controller_test.go

```diff
@@ -405,12 +405,6 @@ var _ = ginkgo.Describe("Workload controller", ginkgo.Ordered, ginkgo.ContinueOn
 							Reason:  "Pending",
 							Message: "By test",
 						}, util.IgnoreConditionTimestampsAndObservedGeneration),
-						gomega.BeComparableTo(metav1.Condition{
```

Comment:
Because this test doesn't execute jobframework reconciler. jobframework reconciler should handle AdmissionCheck eviction and set Requeued condition to true. 

I can add something like this. But I'm not sure that it will give us something.

```go
workload.SetRequeuedCondition(&createdWl, "AdmissionCheck", "At least one admission check is false", true)
gomega.Expect(k8sClient.Status().Update(ctx, &createdWl)).Should(gomega.Succeed())
```

Reply:
No, that's ok.
Thanks for explaining.